bower.json

```diff
@@ -14,6 +14,7 @@
     "jquery": "latest",
     "jquery-deparam": "latest",
     "jquery-turtle": "PencilCode/jquery-turtle#master",
+    "js2coffee": "js2coffee/js2coffee#master",
```

Comment:
js2coffee is not available as a bower package. So, I had to add the tagged version